TEI/ParlaMint-schemaSpecs.editing.odd.xml

```diff
@@ -1,1703 +1,4094 @@
+         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
+            <placeName>
+               <measure>10 miles</measure>
+               <offset>Northeast of</offset>
+               <settlement>Attica</settlement>
+            </placeName>
+         </egXML>
+      </exemplum>
+   </elementSpec>
+   <elementSpec ident="affiliation" module="namesdates" mode="change">
+      <desc xmlns:xi="http://www.w3.org/2001/XInclude"
+            versionDate="2005-01-14"
+            xml:lang="en">contains an informal description of a person's present or past affiliation with some
+    organization, for example an employer or sponsor.</desc>
+      <content>
+         <textNode/>
+      </content>
+      <attList>
+         <attDef ident="role" mode="replace" usage="req">
+            <valList type="closed">
+               <valItem ident="member"/>
+               <valItem ident="president"/>
+               <valItem ident="vicePresident"/>
+               <valItem ident="chairman"/>
+               <valItem ident="viceChairman"/>
+               <valItem ident="minister"/>
+               <valItem ident="MP"/>
+               <valItem ident="substituteMP"/>
+               <valItem ident="associateMember"/>
+               <valItem ident="nonAttachedMember"/>
+               <valItem ident="ministerDelegate"/>
+               <valItem ident="ministerOfState"/>
+               <valItem ident="secretaryOfState"/>
+               <valItem ident="observer"/>
+               <valItem ident="verifier"/>
+               <valItem ident="vicePublicDefenderOfRights"/>
+               <valItem ident="headOfDelegation"/>
+               <valItem ident="presidiumMember"/>
+               <valItem ident="publicDefenderOfRights"/>
+               <valItem ident="alternateOfDelegation"/>
+               <valItem ident="replacement"/>
+               <valItem ident="candidateMP"/>
+               <valItem ident="academician"/>
+               <valItem ident="candidateChairman"/>
+               <valItem ident="chairperson"/>
+               <valItem ident="chiefInspector"/>
+               <valItem ident="commander"/>
+               <valItem ident="constitutionalJudge"/>
+               <valItem ident="deputyChief"/>
+               <valItem ident="deputyMinister"/>
+               <valItem ident="deputyPrimeMinister"/>
+               <valItem ident="director"/>
+               <valItem ident="headOfDepartment"/>
+               <valItem ident="leader"/>
+               <valItem ident="ombudsman"/>
+               <valItem ident="presidentEP"/>
+               <valItem ident="primeMinister"/>
+               <valItem ident="prosecutorGeneral"/>
+               <valItem ident="secretary"/>
+               <valItem ident="secretaryGeneral"/>
+               <valItem ident="viceDirector"/>
+               <valItem ident="speaker"/>
+            </valList>
+         </attDef>
+         <pm:attPreserve ident="key"/>
```

Comment:
```suggestion
```

Reply:
Agreed, doesn't seem to break anything at first glace.